scripts/cursor/rules-for-integration-templates.ts

```
@@ -0,0 +1,37 @@
+
+function main() {
+    const content = [
+        FRONTMATTER,
+        PERSONA_SECTION,
+        fs.readFileSync(PATHS.writing, 'utf-8'),
+        fs.readFileSync(PATHS.templates, 'utf-8'),
+        fs.readFileSync(PATHS.patterns, 'utf-8')
+    ].join('');
+
+    fs.writeFileSync(PATHS.output, content);
+    console.log(`✅ MDC file written to: ${PATHS.output}`);
```

ESLint

[error] Use nango.log instead of console.log

(@nangohq/custom-integrations-linting/no-console-log)